online-shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -0,0 +1,34 @@
+<html>
+
+<head>
+    <title>Online shop</title>
+</head>
+
+<body>
+    <div class = "header">
+        <div class = "header">
+            <h1>Product: {{product?.name}}</h1>
+        </div>
+        <div class = "header">
+            <button>Edit</button>
+            <button id="button_delete">Delete</button>
+        </div>
+    </div>
+    <div class="container_details">
+        <div class="informational_details">Name:</div>
+        <div>{{ product?.name }}</div>
+        <img class="container_image" src="{{product?.image}}"/>
```

Comment:
You're not supposed to use interpolation for the src, you can use square brackets and use the input, such as `[src]="products.image"`